### ISSUE TITLE
Roboto-Regular.woff not present

### ISSUE THREAD

**Issue by PaulL1** [ngbp/ngbp#121]
Another minor issue.  My install gives me an error looking for the font Roboto-Regular, which should be in assets/fonts/Roboto-Regular.woff.

This looks to be included from src/less/main.less, line 40 and below.  

I can see that this can be installed by:
  npm install connect-fonts-roboto
or by:
  bower install roboto-fontface --save-dev
And I think the latter is the preferred one.

I think this could be fixed by editing bower.json as follows:

<pre>
    "angular-ui-utils": "~0.0.3",
+    "roboto-fontface": "~0.1.2"
  },
</pre>

I would do this as a git pull request, but it seems that my skills don't quite extend that far....

**Comment by PaulL1** [comment]
Actually, doing more on this today, and there's more to it than that.  There's also something to do so as to push the font file through into the build directory.  I think the necessary change is to modify build.config.js to add the line:

<pre>
      'vendor/angular-grid/build/ng-grid.js',
+      'vendor/roboto-fontface/fonts/Roboto-Regular.woff'
    ],
</pre>

At least that works, but I sort of feel like that can't really be the right answer - what if someone used bold for example?  I guess the idea is that someone should know whether they did or did not use bold, and include the fonts appropriately in their build file.

**Comment by PaulL1** [comment]
Nope, that was blatantly wrong as well, because that section of the build file is clearly for javascript, and doesn't put things into the assets directory.  Sigh.  I don't see anywhere obvious to add fonts into the build process.  I've copied them manually for now.  It's not that I really need the fonts, I just don't like having errors.  :-)

**Comment by hourback** [comment]
:+1:

**Comment by joshdmiller** [comment]
The `@font-face` declaration should be removed from the less file and this added to `index.html`:

``` html
<link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
```

I'll add this to the next release.

**Comment by PaulL1** [comment]
Ah.  Much better.  I'd assumed you wanted the fonts locally, but this makes more sense for a boilerplate - no point in pulling something locally that it's likely people will change anyway.  I'll cancel my pull request that related to this.

**Comment by PaulL1** [comment]
Man, half an hour of messing with github to do 2 minutes of work!!!  But, I needed to learn my way around github anyway.
